### PR TITLE
Port To Python 3

### PR DESCRIPTION
**Pull request by Creatune**
Worked well before and after porting.

### CODE REVIEW

File: AppelHaken.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def run(self):
                 elif event.type == pygame.MOUSEMOTION:
                     g.pos=event.pos
                     g.redraw=True
-                    if self.canvas<>None: self.canvas.grab_focus()
+                    if self.canvas!=None: self.canvas.grab_focus()
```
Comment (chimosky) [inline AppelHaken.py]: You could space the operators.

File: utils.py [hunk 2]
```diff
@@ -52,7 +52,7 @@ def load_image(file1,alpha=False,subdir=''): # eg subdir='glow'
     try:
         img=pygame.image.load(fname)
     except:
-        print "Peter says: Can't find "+fname; exit()
+        print("Peter says: Can't find "+fname); exit()
```
Comment (chimosky) [inline utils.py]: For consistency, you can move `exit()` to a new line.

**Comment by chimosky** [comment]
Reviewed, works as expected.

**Comment by Creatune** [comment]
Sure, I will make the changes. I will finish this by 7 pm (IST) :)

**Comment by Aniket21mathur** [comment]
@Creatune any progress?

**Comment by quozl** [comment]
The mixing of spacing in function argument lists is distracting; can you stick to the style in use?

**Comment by Creatune** [comment]
> The mixing of spacing in function argument lists is distracting; can you stick to the style in use?

I accidentally erased all the operating systems in my laptop in an attempt to install fedora 31 :'( . I will get back to this as fast as possible

**Comment by srevinsaju** [comment]
Tested on Sugar Debian Live, Sugar DE, no warnings or errors

**Comment by srevinsaju** [comment]
@Creatune I have fixed some more more PEP errors on https://github.com/Creatune/appel-haken-activity/pull/1 @Creatune please review and merge :)

File: utils.py (hunk 2 above)
Comment (Creatune) [reply utils.py]: The semicolon at the end of the line throws me off as python doesn't require semicolons at the end of lines (or so I think). So can anyone please explain why the semicolon is there? Thank you.

Comment (srevinsaju) [reply utils.py]: Python can use semicolons to have many lines in one single lines (one liner codes). But for the sake of PEP8 formatting, it is better not have semicolons;

```
print("Peter says: Can't find "+fname); exit()
```
is better written as 
```
print("Peter says: Can't find "+fname)
exit()
```

**Comment by Creatune** [comment]
It's fun to work with a fellow gci student :)

File: utils.py (hunk 2 above)
Comment (Creatune) [reply utils.py]: Got it. Thanks :)

**Comment by quozl** [comment]
Impractical to review now that we have flake8 fixes mixed with a port.  I've rebased the port commits as b6865f4.  I tried 7e0d3d5 as well, but got continued flake8 warnings;

```log
/utils.py:7:1: F401 'copy' imported but unused
./utils.py:27:12: E711 comparison to None should be 'if cond is None:'
./utils.py:38:12: E711 comparison to None should be 'if cond is None:'
./utils.py:43:5: E722 do not use bare except'
./utils.py:47:5: E722 do not use bare except'
./utils.py:72:5: E722 do not use bare except'
./utils.py:85:9: E722 do not use bare except'
./utils.py:192:5: E722 do not use bare except'
./utils.py:246:80: E501 line too long (84 > 79 characters)
./utils.py:249:16: E711 comparison to None should be 'if cond is None:'
./utils.py:253:25: E711 comparison to None should be 'if cond is not None:'
./AppelHaken.py:22:1: E722 do not use bare except'
./AppelHaken.py:30:80: E501 line too long (84 > 79 characters)
./AppelHaken.py:111:24: E711 comparison to None should be 'if cond is not None:'
./AppelHaken.py:132:36: E711 comparison to None should be 'if cond is not None:'
./load_save.py:12:5: E722 do not use bare except'
./ah.py:72:19: E711 comparison to None should be 'if cond is not None:'
./ah.py:161:17: F841 local variable 'cxy' is assigned to but never used
./ah.py:224:15: E711 comparison to None should be 'if cond is not None:'
./activity.py:67:9: E722 do not use bare except'
./g.py:45:5: F841 local variable 'frame_rate' is assigned to but never used
./buttons.py:121:19: E721 do not compare types, use 'isinstance()'
./buttons.py:135:19: E721 do not compare types, use 'isinstance()'
./sugargame/event.py:202:80: E501 line too long (92 > 79 characters)
```

So I didn't merge that one.  @srevinsaju, if you want, please try that one again.

**Pull request closed without merging**